src/common/Utilities/StartProcess.cpp

```diff
@@ -182,21 +182,21 @@ namespace Acore
 
             return CreateChildProcess([&](child& c) -> int
             {
-                int result;
+                int res;
                 my_child = std::reference_wrapper<child>(c);
 
                 try
                 {
                     c.wait();
-                    result = c.exit_code();
+                    res = c.exit_code();
                 }
                 catch (...)
                 {
-                    result = EXIT_FAILURE;
+                    res = EXIT_FAILURE;
                 }
 
                 my_child.reset();
-                return was_terminated ? EXIT_FAILURE : result;
+                return was_terminated ? EXIT_FAILURE : res;
```

Comment:
Personally, I'd prefer to spell out `result` - is naming it `res` a C++ and/or AzerothCore coding standard or something?

Reply:
Similar comment about the line where `appenderName` was changed to `appendName`

Reply:
@steenburgh me too, but the problem here is that there is already a `result` variable in the outer scope, so this would be shadowing it, which is bad

Reply:
Maybe it could be renamed to `exitCode` instead?

Reply:
@steenburgh good idea, done!